README.md

```diff
@@ -2,7 +2,7 @@
 
 <div align="center">
 
-  <img src="bookstore.png" alt="logo" width="340"  height="auto" />
+  <img src="bookstore.png" alt="logo" width="400"  height="auto" />
```

Comment:
I notice that you have not implemented any of the requirements for this milestone. See the list of them below and please do well to implement them and resubmit fo review

### [Project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/use_redux_in_react_components.md)

1. Connect your store to the main component of your React app.
2. Define a state slice which will contain your books - **_Please ensure you use the `createSlice` function from redux toolkit_**
3. Display your books, received from the slice, in a reusable component
4. Dispatch actions using useDispatch
5. Add a component, which includes:
6. An event handler that adds a book to the books array (with attributes id, title and author)
7. Add a component, which includes:
8. An event handler that removes a book from the books array (by id)
9. Add a component, which includes:
10. An event handler that check status (i.e add a functionality for "Check status" button click)